client/.gitignore

```diff
@@ -28,11 +28,11 @@
 .packages
 .pub-cache/
 .pub/
-/build/
+build/
```

Comment:
since we are duplicating the platform-specific gitignores, does it make more sense to just duplicate this gitignore in the `app-en` dir too? alternatively, does git allow symlinking to a gitignore?

Reply:
Yeah I'll just dupe them...